softwarereview_policies.Rmd

```diff
@@ -157,6 +157,120 @@ Despite our best efforts to support contributed software, errors are the respons
 
 In the unlikely scenario that a contributor of a package requests removal of their package from the suite, we retain the right to maintain a version of the package in our suite for archival purposes.
 
+## Ethics, Data Privacy and Human Subjects Research
+
+rOpenSci tools are used for a variety of purposes, but our focus is on
+tools for research. We expect that tools will enable ethical use by research
+practitioners, who are obligated to adhere to ethical codes such [Declaration of
+Helsinki](https://www.wma.net/policies-post/wma-declaration-of-helsinki-ethical-principles-for-medical-research-involving-human-subjects/)
+and [The Belmont
+Report](https://www.hhs.gov/ohrp/regulations-and-policy/belmont-report/index.html).
+Researchers bear responsibility for their use of software, but software
+developers must consider the ethical use of their products, and developers
+themselves adhere to ethical codes for computer professionals such as those
+expressed by [IEEE](https://www.computer.org/education/code-of-ethics) and
+[ACM](https://ethics.acm.org/). rOpenSci contributors often play both roles.
```

Comment:
Maybe re-state explicitely here what the two roles are even if it's written at the beginning of the paragraph?

Reply:
👍